contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -347,6 +373,169 @@ UninstallExtendedHooks(void)
  * 			Hook Functions
  *****************************************/
 
+void pltsql_CreateFunctionStmt(ParseState *pstate,
```

Comment:
Nit: missing required line break after `void`. Also, the function needs to be `static`.